pytensor/graph/basic.py

```diff
@@ -907,6 +907,62 @@ def graph_inputs(
     yield from (r for r in ancestors(graphs, blockers) if r.owner is None)
 
 
+def required_graph_inputs(
+    graph: Variable[Any, Any] | Iterable[Variable[Any, Any]],
+) -> list[Variable[Any, Any]]:
+    """
+    Get the inputs into PyTensor variables
+
+    Parameters
+    ----------
+        graph: PyTensor `Variable` instances
+        Output `Variable` instances from which to search backward through
+        owners.
+
+    Returns
+    -------
+        List of tensor variables that are input nodes with no owner, in the order
+        found by a left-recursive depth-first search started at the nodes in `graphs`.
+
```

Comment:
These docstrings are not well formatted. @OriolAbril can we point to the same docs as for PyMC for PyTensor docs?

Reply:
not sure what you mean, link to numpydoc? https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
I referred the format that pymc uses for [logp](https://github.com/pymc-devs/pymc/blob/6761c0c73ba07cc9dc51ec3adab7f1aa5f76b23d/pymc/logprob/basic.py#L123). Does it seem right?

Reply:
> not sure what you mean, link to numpydoc? https://numpydoc.readthedocs.io/en/latest/format.html

I was asking if we were using a style in PyTensor. I think you answered it

Reply:
oh, yes, same style as in pymc, numpydoc